tests/jerry/func-decl.js

```diff
@@ -38,3 +38,18 @@ check_syntax_error ("'use strict'; function arguments () {}");
 check_syntax_error ("'use strict'; var l = function arguments () {}");
 
 check_syntax_error ("function f__strict_mode_duplicate_parameters (p, p) { 'use strict'; }");
+
+function test_strict_mode_propagation_in_func_expr_and_getters_setters () {
```

Comment:
So are this test fails with this PR? If so then shouldn't this be inside one of the tests/jerry/fail/* directory to test that it fails correctly? Or did I misunderstood something?